snare/cloner.py

```diff
@@ -31,12 +31,18 @@ def __init__(self, root, max_depth, css_validate):
 
     @staticmethod
     def add_scheme(url):
+        test_string = str(url)
+        if test_string[-1] == '/':
+            b_url = yarl.URL(url)
+        else:
+            b_url = yarl.URL(url + '/')
+        url = b_url.human_repr()
```

Comment:
Keep the url as a string, no need of conversion to URL object and then just add the trailing space.